code/modules/clothing/gloves/color.dm

```diff
@@ -80,6 +80,16 @@
 	. = ..()
 	siemens_coefficient = pick(0,0,0,0.5,0.5,0.5,0.75)
 
+/obj/item/clothing/gloves/cut
+	desc = "These gloves would protect the wearer from electric shock.. if the fingers were covered."
```

Comment:
```suggestion
	desc = "These gloves would protect the wearer from electric shock...if the fingers were covered."
```

Reply:
Jared's suggestion needs a space after the ellipsis

Reply:
```suggestion
	desc = "These gloves would protect the wearer from electric shock... if the fingers were covered."
```